tezos/new_context/src/working_tree/working_tree.rs

```diff
@@ -1055,20 +1065,20 @@ impl WorkingTree {
         }
     }
 
-    // TODO: part of the outer layer, this should not be on the working tree
-    fn get_commit(&self, hash: &EntryHash) -> Result<Commit, MerkleError> {
-        match self.get_entry_from_hash(hash)? {
-            Entry::Commit(commit) => Ok(commit),
-            Entry::Tree(_) => Err(MerkleError::FoundUnexpectedStructure {
-                sought: "commit".to_string(),
-                found: "tree".to_string(),
-            }),
-            Entry::Blob(_) => Err(MerkleError::FoundUnexpectedStructure {
-                sought: "commit".to_string(),
-                found: "blob".to_string(),
-            }),
-        }
-    }
+    // // TODO: part of the outer layer, this should not be on the working tree
+    // fn get_commit(&self, hash: &EntryHash) -> Result<Commit, MerkleError> {
```

Comment:
These methods that just read from the index are still on `WorkingTree` but in reality are methods on the index. On the branch where I am fixing the RPCs I am planning to remove these from here.